src/fr/bigsolo/src/eu/kanade/tachiyomi/extension/fr/bigsolo/BigSolo.kt

```
@@ -0,0 +1,266 @@
+        currentSearchQuery = ""
+        return GET("$baseUrl$CONFIG_ENDPOINT", headers)
+    }
+
+    override fun popularMangaParse(response: Response): MangasPage {
+        return searchMangaParse(response)
+    }
+
+    override fun popularMangaFromElement(element: Element): SManga {
+        throw UnsupportedOperationException()
+    }
+
+    override fun popularMangaSelector() = throw UnsupportedOperationException()
+
+    override fun popularMangaNextPageSelector(): String? {
+        throw UnsupportedOperationException()
+    }
+
+    // Latest
+    override fun latestUpdatesRequest(page: Int): Request {
+        throw UnsupportedOperationException()
+    }
+
+    override fun latestUpdatesFromElement(element: Element): SManga {
+        throw UnsupportedOperationException()
+    }
+
+    override fun latestUpdatesSelector(): String {
+        throw UnsupportedOperationException()
+    }
+
+    override fun latestUpdatesNextPageSelector(): String? {
+        throw UnsupportedOperationException()
+    }
+
+    // Search
+    override fun searchMangaRequest(page: Int, query: String, filters: FilterList): Request {
+        currentSearchQuery = query
+        return GET("$baseUrl$CONFIG_ENDPOINT", headers)
+    }
+
+    override fun searchMangaParse(response: Response): MangasPage {
+        val config = parseConfigResponse(response.body.string())
+        val mangaList = mutableListOf<SManga>()
+
+        for (fileName in config.localSeriesFiles) {
+            try {
+                val seriesData = fetchSeriesData(fileName)
+
+                // Filter by title if a query is provided
+                if (currentSearchQuery.isBlank() || seriesData.title.contains(
+                        currentSearchQuery,
+                        ignoreCase = true,
+                    )
+                ) {
+                    mangaList.add(seriesData.toSManga())
+                }
+            } catch (e: Exception) {
```

don't swallow exceptions